servers/display_server.cpp

```diff
@@ -709,6 +727,12 @@ void DisplayServer::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("screen_get_scale", "screen"), &DisplayServer::screen_get_scale, DEFVAL(SCREEN_OF_MAIN_WINDOW));
 	ClassDB::bind_method(D_METHOD("is_touchscreen_available"), &DisplayServer::is_touchscreen_available, DEFVAL(SCREEN_OF_MAIN_WINDOW));
 	ClassDB::bind_method(D_METHOD("screen_get_max_scale"), &DisplayServer::screen_get_max_scale);
+
+	ClassDB::bind_method(D_METHOD("ios_set_gesture_state", "ios_gesture_set"), &DisplayServer::ios_set_gesture_state);
+	ClassDB::bind_method(D_METHOD("ios_get_gesture_state"), &DisplayServer::ios_get_gesture_state);
+	ClassDB::bind_method(D_METHOD("ios_set_status_bar_appearance", "hide_status_bar", "status_bar_fade_time"), &DisplayServer::ios_set_status_bar_appearance, DEFVAL(false), DEFVAL(0.2));
+	ClassDB::bind_method(D_METHOD("ios_get_status_bar_appearance"), &DisplayServer::ios_get_status_bar_appearance);
```

Comment:
I'm not sure about functions and constants with `ios` in the name, are there equivalents for Android? Even if it's only relevant for iOS now, it's probably better to keep API OS-agnostic.

Reply:
As far as I am aware, the only equivalent android has is Immersive Mode which is just a true/false setting that hides both and requires a double swipe all in one.